packages/cadl-ranch-specs/http/client/structure/client-operation-group/client.tsp

```diff
@@ -54,14 +54,14 @@ namespace Client.Structure.ClientOperationGroup {
     name: "SubNamespace.SecondClient",
     service: Client.Structure.Service,
   },
-  "csharp,java,python"
+  "csharp,java"
 )
 @client(
   {
     name: "SecondClient",
     service: Client.Structure.Service,
   },
-  "javascript,go"
+  "javascript,go,python"
```

Comment:
Python SDK doesn't permit client name has namespace prefix.

Reply:
@tadelesh @iscai-msft please help review

Reply:
> Python SDK package name, namespace and package folder are one-one mapping. If add sub-namespace, package folder shall also have related change which makes the folder complicated to understand for users.

do you have a link to the generated code?

Reply:
Without this PR, python emitter will crash for this case.

Reply:
approve this change, but i believe we need to discuss this scenario in python.

Reply:
this is related to the overarching namespace changes we're looking into, see @srnagar 's proposal [here](https://gist.github.com/srnagar/a695faa1a4d8a5dbf44af826f05916f0). Python should start following this structure, which is something that all languages will start working on

Reply:
I see and create https://github.com/Azure/autorest.python/issues/2795 to track